src/routers/inviter_router.py

```diff
@@ -9,4 +10,16 @@ def create_invitations_list():
 
 @router.get("/invitations", summary="Gets all the invitations for this user.")
 def get_invitations(invitations: InvitationRepository = Depends(create_invitations_list), user_id: str = None):
-    return invitations.query(lambda x: x.filter(Invitation.user_id == user_id))
\ No newline at end of file
+    return invitations.query(lambda x: x.filter(Invitation.user_id == user_id))
+
+@router.get("/invite", summary="Creates a new invitation for this user.")
+def create_invitation(invitations: InvitationRepository = Depends(create_invitations_list), user_id: str = None, invitees_amount: int = None, invitees_arrival_timestamp_epoch: int = None,comment_for_guard: str =  None):
+    new_invitation =  Invitation(
```

Comment:
Do some data validations

Reply:
Why are you using get to create new invite?
Use post or put.
Also, prefably, don't use query parameters for input.
Use the body